server-mongo/changeDatabase.jq

```diff
@@ -3,9 +3,9 @@
   .realm.provider = "mongo" | 
   .user.provider = "mongo" |
   .userSessionPersister.provider = "mongo" |  
-  .connectionsMongo.default.host = "${env.MONGO_PORT_27017_TCP_ADDR}" |
-  .connectionsMongo.default.port = "27017" |
-  .connectionsMongo.default.db = "${env.MONGODB_DBNAME:keycloak}" |
+  .connectionsMongo.default.host = "${env.MONGO_HOST}" |
```

Comment:
The above is an example of the env vars auto-generated when using docker linking :( Converting to compose allows us to clean them up :)